bridge/Mixer.cpp

```diff
@@ -1111,10 +1108,14 @@ bool Mixer::configureVideoStream(const std::string& endpointId,
         videoStream->_secondarySimulcastStream = secondarySimulcastStream;
     }
 
-    if (absSendTimeExtensionId.isSet())
+    videoStream->_rtpMap._absSendTimeExtId = absSendTimeExtensionId;
+    if (videoStream->_rtpMap._absSendTimeExtId.isSet())
+    {
+        videoStream->_transport->setAbsSendTimeExtensionId(videoStream->_rtpMap._absSendTimeExtId.get());
```

Comment:
We should post a single Job to set rtxprobeSsrc, rtxpayloadtype, and absendtimeext hdr id

Reply:
I updated the PR with setting the payload type together with the RTX probe source, but I'm leaving the abssendtime as out of scope of this PR since it also affects audio only endpoints and therefore requires some more thought.